test/images/image-util.sh

```diff
@@ -284,8 +284,8 @@ if [[ "${WHAT}" == "all-conformance" ]]; then
   shift
   conformance_images=("busybox" "agnhost" "jessie-dnsutils" "kitten" "nautilus" "nonewprivs" "resource-consumer" "sample-apiserver")
   for image in "${conformance_images[@]}"; do
-    eval "${TASK}" "${image}" "$@"
```

Comment:
Just drop eval - not needed

Reply:
Did not expect that to just work. TIL

Reply:
I feel sort of guilty teaching anyone how to use bash.